net-analyzer/gvmd/Manifest

```diff
@@ -1 +1,2 @@
 DIST gvmd-21.4.5.tar.gz 1041186 BLAKE2B 18edc375084a6c6445f12779935e34d20826d2390481cbcdc06cab8dea81ea9c6bddeda437b081137f858e860f725431d33d5654eae231e303e0dc5618a89c8a SHA512 5b22b9bfd58ae73fb973941c7a96027599ccc5b78d822eac9c840460cb75bf60a398519569daa04d5c4d935dbe3974335f87e2c1c5fa751b4cec1ea6e5f064ec
```

Comment:
The commit message here isn't very descriptive.. "to fix bug 21.4.5-r5" ok how? what was wrong with it?

Reply:
also, in this commit, you're adding a new distfile for 22.4.2 which I assume is a mistake